.gitignore

```diff
@@ -54,3 +54,5 @@ dkms.conf
 # Tmp files
 *.*~
 *~
+lib/python/Pyxsim/__pycache__/
```

Comment:
**/__pycache__

Reply:
Neat, didn't realise that existed - changed.